install.yaml

```diff
@@ -71,6 +71,7 @@ pre_install_actions:
       read platform_project
       echo "platform_project = '${platform_project}'"
       # Put the platform_project in to the project's web environment
+      cd ${DDEV_APPROOT}
```

Comment:
I thought it would not make impact on `ddev config`. So more add-ons will be affected.

Given this, maybe changing the `pre_install_actions` context to `.ddev` is too disruptive?

(There is one more `ddev config` inside this yaml, that call `ddev config`, that you didn't change.)